src/theme/variables.scss

```diff
@@ -89,8 +91,8 @@
   }
 
   ion-tab-button {
-    --color: var(--ion-color-primary-shade);   
```

Comment:
@sapphaos @Kzamania Please set `files.trimTrailingWhitespace` to true (in VSCode settings) to prevent this in the future. Ideally, we should have used an `.editorconfig` file to have the same coding style at the beginning of the project

Reply:
Done.